changelogs/fragments/406-lldp-tlv-support.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - sonic_lldp - Enable PVID, Vlan name, Link aggregation, Max-Frame-Size TLV support in LLDP(https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/406).
```

```suggestion
  - sonic_lldp - Enable PVID, Vlan name, Link aggregation, Max-Frame-Size TLV support in LLDP (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/406).
```
I recommend adding a space between the description and the link to the PR

---

addressed